### PR TITLE
fix: spelling and sort languages in alphabetical order

### LINKED ISSUE #5347: Footer/Internationalization: Show languages dropdown as written in their original way
In the footer we have a language dropdown that shows all languages in English. Expected is that languages show up as they are written in their own language, e.g. German should be Deutsch, Vietnamese should be Tiếng Việt etc.
Screenshot from 2020-10-21 19-32-24

### PR DESCRIPTION
**Pull request by Soumi150**
Fixes #5347 

#### Short description of what this resolves:
Fixes spelling of some languages and sorts them in alphabetical order

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

Screenshot (15)

### CODE REVIEW

File: app/services/l10n.js [hunk 1]
```diff
@@ -14,21 +14,21 @@ export default class L10nService extends L10n {
   @computed(function() {
     return {
       'bn'      : 'বাংলা',
-      'zh_Hans' : '中文 (简化版)',
-      'zh_Hant' : '中文 (傳統的)',
+      'zh_Hans' : '中文（简体）',
```
Comment (iamareebjamal) [inline app/services/l10n.js:17]:
```suggestion
      'zh_Hans' : '中文（简体',
```

Comment (iamareebjamal) [inline app/services/l10n.js:17]:
```suggestion
      'zh_Hans' : '中文 (简体)',
```

File: app/services/l10n.js [hunk 2]
```diff
@@ -14,21 +14,21 @@ export default class L10nService extends L10n {
...
+      'zh_Hant' : '中文（繁體）',
```
Comment (iamareebjamal) [inline app/services/l10n.js:18]:
```suggestion
      'zh_Hant' : '中文 (繁體)',
```

File: app/services/l10n.js [hunk 3]
```diff
@@ -14,21 +14,21 @@ export default class L10nService extends L10n {
...
+      'id'      : 'Bahasa Indonesia',
       'ja'      : '日本語',
-      'ru'      : 'русский'
-    };
+      'ko'      : '한국어',
+      'pl'      : 'Polski',
+      'ru'      : 'Русский',
+      'th'      : 'ไทย',
+      'vi'      : 'Tiếng Việt',
+      };
```
Comment (niranjan94) [inline app/services/l10n.js:31]: Codacy found an issue: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403748)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403748&bid=20757933)

File: app/services/l10n.js [hunk 4]
```diff
@@ -14,21 +14,21 @@ export default class L10nService extends L10n {
...
       'hi'      : 'हिंदी',
+      'id'      : 'Bahasa Indonesia',
       'ja'      : '日本語',
-      'ru'      : 'русский'
-    };
+      'ko'      : '한국어',
+      'pl'      : 'Polski',
+      'ru'      : 'Русский',
+      'th'      : 'ไทย',
+      'vi'      : 'Tiếng Việt',
```
Comment (niranjan94) [inline app/services/l10n.js:30]: Codacy found an issue: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403748)

File: app/services/l10n.js [hunk 5]
```diff
@@ -14,20 +14,20 @@ export default class L10nService extends L10n {
   @computed(function() {
     return {
       'bn'      : 'বাংলা',
-      'zh_Hans' : '中文 (简化版)',
-      'zh_Hant' : '中文 (傳統的)',
+      'zh_Hans' : '中文（简体)',
+      'zh_Hant' : '中文（繁體)',
```
Comment (iamareebjamal) [inline app/services/l10n.js:18]: This is present last in WikiPedia. Which sorting have you used?

File: app/services/l10n.js [hunk 6]
```diff
@@ -14,20 +14,20 @@ export default class L10nService extends L10n {
...
-      'pl'      : 'Polskie',
       'es'      : 'Español',
-      'th'      : 'ไทย',
-      'vi'      : 'Tiếng Việt',
+      'fr'      : 'Français',
       'hi'      : 'हिंदी',
+      'id'      : 'Bahasa Indonesia',
       'ja'      : '日本語',
-      'ru'      : 'русский'
+      'ko'      : '한국어',
```
Comment (iamareebjamal) [inline app/services/l10n.js:26]: Korean is last in Wikipedia list as well

**Comment by Soumi150** [comment]
I have sorted in alphabetical order .For example Chinese after Bengali.

**Comment by iamareebjamal** [comment]
Yes, that's mentioned in the issue comment

File: app/services/l10n.js (hunk 6 above)
Comment (Soumi150) [reply app/services/l10n.js]: Korean is not last in wikipedia ,its after Francais and Galego.

File: app/services/l10n.js [hunk 7]
```diff
@@ -14,20 +14,20 @@ export default class L10nService extends L10n {
...
-      'fr'      : 'français',
-      'de'      : 'Deutsche',
-      'id'      : 'bahasa Indonesia',
-      'ko'      : '한국어',
-      'pl'      : 'Polskie',
       'es'      : 'Español',
-      'th'      : 'ไทย',
-      'vi'      : 'Tiếng Việt',
+      'fr'      : 'Français',
+      'ko'      : '한국어',
```
Comment (iamareebjamal) [inline app/services/l10n.js:21]: Korean is last https://www.wikipedia.org/

File: app/services/l10n.js [hunk 8]
```diff
@@ -14,20 +14,20 @@ export default class L10nService extends L10n {
...
       'hi'      : 'हिंदी',
+      'id'      : 'Bahasa Indonesia',
       'ja'      : '日本語',
-      'ru'      : 'русский'
+      'pl'      : 'Polski',
+      'ru'      : 'Русский',
+      'th'      : 'ไทย',
+      'vi'      : 'Tiếng Việt',
+      'zh_Hans' : '中文（简体)',
+      'zh_Hant' : '中文（繁體)',
```
Comment (niranjan94) [inline app/services/l10n.js:30]: Codacy found an issue: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403748)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403748&bid=20757933)

File: app/services/l10n.js (hunk 7 above)
Comment (Soumi150) [reply app/services/l10n.js]: Sorry, I was referring to another wikipedia page, changed it now.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**